photon-lib/src/main/java/org/photonvision/PhotonCamera.java

```diff
@@ -346,20 +353,28 @@ private void verifyVersion() {
         // Heartbeat entry is assumed to always be present. If it's not present, we
         // assume that a camera with that name was never connected in the first place.
         if (!heartbeatEntry.exists()) {
-            Set<String> cameraNames = cameraTable.getInstance().getTable(kTableName).getSubTables();
+            var cameraNames = getTablesThatLookLikePhotonCameras();
             if (cameraNames.isEmpty()) {
                 DriverStation.reportError(
-                        "Could not find any PhotonVision coprocessors on NetworkTables. Double check that PhotonVision is running, and that your camera is connected!",
+                        "Could not find **any** PhotonVision coprocessors on NetworkTables. Double check that PhotonVision is running, and that your camera is connected!",
                         false);
             } else {
                 DriverStation.reportError(
                         "PhotonVision coprocessor at path "
                                 + path
                                 + " not found on NetworkTables. Double check that your camera names match!",
                         true);
+
+                var cameraNameStr = new StringBuilder();
+                for (var c : cameraNames) {
+                    cameraNameStr.append(" ==> ");
+                    cameraNameStr.append(c);
+                    cameraNameStr.append("\n");
+                }
+
                 DriverStation.reportError(
-                        "Found the following PhotonVision cameras on NetworkTables:\n"
-                                + String.join("\n", cameraNames),
+                        "Found the following PhotonVision cameras active on NetworkTables:\n"
```

Comment:
They aren't necessarily active cause they are on NT, just present. I think this should be left as is.